src/shared/utils/get-custom-tags.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+export const getCustomTags = async (domain:string) => {
+
+    //const url = `https://mediajel-tracker-custom-tags-staging.s3.amazonaws.com/`;
+    const url = `https://d3wuj95q2emo9.cloudfront.net/${domain}.js`
+    try {
+      const response = await fetch(url);
+      if (!response.ok) {
+        throw new Error(`HTTP error! status: ${response.status}`);
+      }
```

Comment:
Ideally we'd want to console.warn that if the response status code is `404` or the JS tag is not found it'd say something like

```
custom tag for {window.location.hostname} not found
```

Otherwise all of our deployed tags without custom js tags associated with them in the s3 bucket would have the `HTTP error...`

Ofcourse if it's any other error than a `not found` error. We'd want to log it to the console.

Reply:
removed throw error and added proper console.error message